style.css

```diff
@@ -0,0 +1,132 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Serif+Display:ital,wght@1,500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Heebo:wght@300&display=swap');
+
+:root{
+    --background-color: #F8F9FA !important;
```

Comment:
If you can just avoid using !important.
>Using !important is a bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets.

Reply:
Yes, but I used my component library so I needed to override those colors